dhcpv4/ztpv4/parse_circuitid_test.go

```diff
@@ -18,7 +18,8 @@ func TestMatchCircuitID(t *testing.T) {
 		{name: "juniperQFX pattern", circuit: "et-0/0/0:0.0", want: &CircuitID{Slot: "0", Module: "0", Port: "0", SubPort: "0"}},
```

Comment:
I realize now that the test should be grouped with this pattern instead.

Reply:
You mean since we have "juniperQFX pattern",  we dont need the other two tests "Juniper QFX pattern et" and ""Juniper QFX pattern xe"?

Reply:
There are 2 Juniper QFX patterns:

1) https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/ztpv4/parse_circuitid.go#L21
2) https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/ztpv4/parse_circuitid.go#L30

This grouping is mixing both patterns and making this a bit more confusing.

I think the intention was to add the test close to the other one.